src/commands/remote_signer_setup.py

```diff
@@ -70,7 +71,23 @@
     envvar='LOG_LEVEL',
     help='The log level.',
 )
+@click.option(
+    '--dappnode',
+    help='Add fields required by Dappnode Staking Brain to the import request (tags and fee recipients)',
+    envvar='DAPPNODE',
+    is_flag=True,
+)
+@click.option(
+    '--execution-endpoints',
+    type=str,
+    envvar='EXECUTION_ENDPOINTS',
+    prompt='Enter comma separated list of API endpoints for execution nodes. Used to retrieve vault validator fee recipient (only needed if flag --dappnode is set).',
```

Comment:
Too much verbosity for prompt. I suggest to put this message into 'help' section. Prompt should stay simple:
> Enter comma separated list of API endpoints for execution nodes

Reply:
You are right and, actually, as `dappnode` command must be non-interactive, it does not make sense to have the prompt field, so I have just left the `help` field